src/main/java/Duke.java

```diff
@@ -1,10 +1,283 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+
 public class Duke {
+    private boolean isActive;
+    private List<Task> myList;
+
+    public Duke() {
+        this.isActive = true;
+        this.myList = new ArrayList<>();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Duke ft = new Duke();
+        ft.init();
+    }
+    public void init() {
+        System.out.println("    ____________________________________________________________\n"
+                + "    Hello! I'm FriendlyTool\n"
+                + "    What can I do for you?\n"
+                + "    ____________________________________________________________"
+        );
+        try {
+            loadTask();
+        } catch (ftException e) {
+            System.out.println(e.getMessage());
+        }
+        Scanner sc = new Scanner(System.in);
+        while (this.isActive) {
+            String input = sc.nextLine();
+            try {
+                nextAction(input);
+            } catch (ftException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+        System.out.println("    ____________________________________________________________\n"
+                + "    Bye. Hope to see you again soon!\n"
+                + "    ____________________________________________________________" );
+    }
+
+    private void nextAction(String input) throws ftException {
```

Comment:
While `nextAction` does imply a continuation of interaction with the user, something like `handleUserResponse` might be better as it makes it clear who is taking the next action (the bot in this case) and who it is meant for (the user)